src/elevation.json

```diff
@@ -0,0 +1,25 @@
+{
+  "elevation": {
+    "hidden": {
+      "value": "-1"
+    },
```

Comment:
What is this value to be used for?

Reply:
Brandon and I didn't really know what it meant but we just copied and pasted it from another file😅 if it's not needed, I will delete it!

Reply:
I don't think it's needed in this context. I would suggest removing it.